install/src/HieInstall.hs

```diff
@@ -111,17 +109,15 @@ defaultMain = do
       )
 
     -- cabal specific targets
-    phony "cabal-build-latest" (need ["cabal-hie-" ++ last ghcVersions])
-    phony "cabal-build"  (need ["build-data", "cabal-build-latest"])
-    phony "cabal-build-data" $ do
+    when isRunFromCabal $ phony "cabal-build-latest" (need ["cabal-hie-" ++ last ghcVersions])
```

Comment:
Same for cabal based targets:
```haskell
when isRunFromCabal $ do
  phony "cabal-build-latest" (need ["cabal-hie-" ++ last ghcVersions])
  phony "cabal-build"  (need ["build-data", "cabal-build-latest"])
  -- .....
```

or even better:
```haskell
phony (buildSystem ++ "-build-latest") (need [buildsystem ++ "-hie-" ++ last ghcVersions])
phony (buildSystem ++ "-build")  (need ["build-data",buildsystem ++ "-build-latest"])
-- etc
```

Reply:
Or take ride of prefixed targets entirily and simple executing the targets with one tool or another